src/pages/components/loading/style.mdx

```diff
@@ -8,11 +8,11 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 ## Color
 
-| Class                  | Property         | Color token         |
-| ---------------------- | ---------------- | ------------------- |
-| `.bx--loading__svg`    | stroke           | `$interactive-04`   |
-| `.bx--loading--small`  | stroke           | `$interactive-04`   |
-| `.bx--loading-overlay` | background-color | `rgba($ui-02, 0.6)` |
+| Class                  | Property         | Color token    |
+| ---------------------- | ---------------- | -------------- |
+| `.bx--loading__svg`    | stroke           | `$interactive` |
+| `.bx--loading--small`  | stroke           | `$interactive` |
+| `.bx--loading-overlay` | background-color | `$overlay'     |
```

Comment:
You've got the wrong `'` on the send of overlay so the visual style isn't showing up correctly. 
![image](https://user-images.githubusercontent.com/11670886/118710156-8a426b80-b7e3-11eb-9b5c-d7e65855ed53.png)

Reply:
```suggestion
| `.bx--loading-overlay` | background-color | `$overlay`     |
```